matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/crypto/MXOlmDevice.kt

```diff
@@ -421,6 +428,34 @@ internal class MXOlmDevice @Inject constructor(
         return null
     }
 
+    fun storeOutboundGroupSessionForRoom(roomId: String, sessionId: String) {
+        outboundGroupSessionStore[sessionId]?.let {
```

Comment:
You should probably update the comment above `outboundGroupSessionStore` declaration:
> The outbound group session.
They are not stored in 'store' to avoid to remember to which devices we sent the session key.
Plus, in cryptography, it is good to refresh sessions from time to time.

So this assertion is not true anymore?

Reply:
updated